### ISSUE TITLE
Zoom limit feature from the Gyroflow nightly doesn't work with the plugin

### ISSUE THREAD

**Issue by Shadetail** [gyroflow/gyroflow-ofx#63]
Zoom limit feature from the Gyroflow nightly doesn't work with the plugin, result is that when zoom limit is used the result of smoothing in Davinci looks very different from the result in the Gyroflow standalone, in my case it was drastically more zoomed in.

The zoom limit feature is from this Gyroflow issue: 
[Dynamic Smothness - zoom to "not more than XX%"](https://github.com/gyroflow/gyroflow/issues/658)

It's such a game changing feature, to the point where it almost makes the plugin obsolete if the plugin doesn't support it. Please add the support for it! 🙏

**Comment by AdrianEddy** [comment]
Are you sure you used latest dev build of the plugin? I tested it and it worked. You need to load the project file

**Comment by Shadetail** [comment]
Yeah, I just doublechecked  to make sure, I deleted the  C:\Program Files\Common Files\OFX\Plugins\Gyroflow.ofx.bundle folder, confirmed that Gyroflow disappeared from Davinci Resolve, then downloaded the new one from [this link.](https://nightly.link/gyroflow/gyroflow-ofx/workflows/build/main/gyroflow-ofx-windows.zip) found on https://gyroflow.xyz/devbuild

I can see that the .gyroflow project file contains:
    "max_zoom": 100.0,
    "max_zoom_iterations": 20

And in davinci resolve I have Gyroflow applied to video and I'm clicking on "Load for current file" button. I can see that the value from the .gyroflow file is loaded as FOV in the plugin menu gets set to 1.078 after clicking the "Load for current file" button.

And when I place the Gyroflow and Davinci on the same frame of the video, I can see that video in Davinci is more cropped in.

Is there something I'm supposed to be doing differently?

Here is the entire content of the .gyroflow project file in case that's useful: 
[project-file.zip](https://github.com/user-attachments/files/16203208/project-file.zip)

I can upload the video as well if needed, though the one this example file references is 11 GB, but I could probably find a smaller one if needed.
